packages/vue-components/src/utils/utils.js

```diff
@@ -159,3 +159,74 @@ export function VueFixer (vue) {
   vue.mixins.unshift(mixin)
   return vue
 }
+
+// Used in the TipBox component to classify the different styles used by bootstrap from the user input.
+// @params (the user input type of the box)
+export function classifyBootstrapStyle(type, theme) {
+  const defaultStyles
+    = ['warning', 'info', 'definition', 'success', 'danger', 'tip', 'important', 'wrong'];
+  const colorStyles
+    = ['primary', 'secondary', 'success', 'danger', 'warning', 'info', 'light', 'dark', 'muted', 'white'];
```

Comment:
iirc `muted / white` are only available for bootstrap text colors, [alerts](https://getbootstrap.com/docs/4.0/components/alerts/) don't have this scheme

Reply:
Ok will remove the options for muted and white then!